src/components/periodic-table-page/PeriodicTablePage.tsx

```diff
@@ -10,102 +8,90 @@ import PeriodicTable, {
   IPeriodicTableElement,
 } from "../periodic-table/PeriodicTable";
 import PtElementInfo from "../pt-element/PtElementInfo";
-import Modal from "../shared/modal/Modal";
 import Navbar from "../shared/navbar/Navbar";
 import SwipeableModal from "../shared/swipeable-modal/SwipeableModal";
 import ElementInfo from "./element-info/ElementInfo";
 import "./PeriodicTablePage.scss";
 
-type Props = RouteComponentProps<any> & React.Props<any>;
-
 interface IPeriodicTablePageState {
   elementInfo: {
     element: IElement;
     open: boolean;
   };
 }
 
-@autobind
-class PeriodicTablePage extends React.Component<
-  Props,
-  IPeriodicTablePageState
-> {
-  public state: IPeriodicTablePageState = {
+function PeriodicTablePage() {
+  const history = useHistory();
+
+  const [state, setState] = React.useState<IPeriodicTablePageState>(() => ({
     elementInfo: {
       element: ElementManager.getElement(1) as IElement,
       open: false,
     },
-  };
```

Comment:
generally we would name the state when using `useState` instead of just using `state` and `setState`.  In this case, I don't thing the `elementInfo` layer of the state is actually giving us anything so I suggest using that:

```ts
  const [elementInfo, setElementInfo] = React.useState<IPeriodicTablePageState>(() => ({
    element: ElementManager.getElement(1) as IElement,
    open: false,
  }));
```